app/views/content/service_privacy_notice_candidate.html.erb

```diff
@@ -228,21 +227,27 @@
 
     <h3 class="govuk-heading-s">Sharing data with providers</h3>
 
-    <p class="govuk-body">We share teacher training candidate and referee data with the providers to which the
+    <p class="govuk-body">We share data about candidates and people who give references with the providers to which the
       candidate has applied.</p>
 
     <p class="govuk-body">We have a data sharing agreement with providers so that they are clear about their
       responsibilities as data controllers in their own right. As data controllers they are required to comply with the
       UK GDPR and DPA2018 including to inform you through a privacy notice like this one, as to what they will be doing
       with your data and your information rights.</p>
 
-    <h3 class="govuk-heading-s">Sharing candidate data with referees</h3>
+    <h3 class="govuk-heading-s">Sharing candidate data with people who give references</h3>
 
-    <p class="govuk-body">We’ll share candidate names with referees so that each referee can identify the candidate and
-      give a reference.</p>
-
-    <p class="govuk-body">We’ll share additional information to help the referee identify the candidate, such as how
-      the candidate knows the referee.</p>
+    <p class="govuk-body">We share candidate data with people who give references, to help them identify candidates and understand how references will be used.</p>
+    
+    <p class="govuk-body">The data we share includes:</p> 
+      
+    <ul class="govuk-list govuk-list--bullet">
+  
+      <li>the candidate’s name</li>
+      <li>the name of the training provider which the candidate has accepted an offer from</li>
+      <li>how the candidate said they know the person giving a reference and how long they’ve known them</li>
+      
+    </ul>
```

Comment:
Could we describe this as 'information' instead of 'data'? Also can we be more definitive rather than using 'includes'?

```suggestion
    <p class="govuk-body">We share candidate information with people who give references, to help them identify candidates and understand how references will be used.</p>
    
    <p class="govuk-body">The information we share is:</p> 
      
    <ul class="govuk-list govuk-list--bullet">
  
      <li>the candidate’s name</li>
      <li>the name of the training provider which the candidate has accepted an offer from</li>
      <li>how the candidate said they know the person giving a reference and how long they’ve known them</li>      
    </ul>
```

Reply:
I agree with both of these points, but I'm trying to make as few changes as possible. Really the privacy policy needs a complete overhaul, including having separate policies for the candidate and provider services.

I've stuck with 'data' since that's what's used in other headings in the same section (for example, 'Sharing data with providers').  Similarly other lists of data use 'includes' in the lead-in. I agree it would be clearer to say 'is' but for now I'd prefer to be consistent.